code/game/gamemodes/cult/runes.dm

```diff
@@ -729,52 +542,33 @@ var/list/teleport_other_runes = list()
 		C.apply_damage(2, BRUTE, pick("l_arm", "r_arm"))
 
 
-//Rite of the Unheard Whisper:  Deafens all non-cultists nearby.
+//Rite of the Unholy Curse:  Deafens, blinds and mutes all non-cultists nearby.
 /obj/effect/rune/deafen
-	cultist_name = "Deafen"
-	cultist_desc = "Causes all non-followers nearby to lose their hearing."
+	cultist_name = "Curse"
+	cultist_desc = "Causes all non-followers nearby to lose their hearing, sight and voice."

```

Comment:
Theme; call it Debilitate with the admittedly kind of silly secondary name 'Rite of the Shadowed Mind'